Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -958,6 +958,7 @@ def list_agents_command(client: Client, args: dict) -> CommandResults:
         scan_status=args.get('scan_status'),
         os_type=args.get('os_type'),
         created_at=args.get('created_at'),
+        limit=args.get('limit'),
```

Comment:
Does the API recieve an int as the limit or a string?

Reply:
yes I have change the code to cast to int